game-of-life-native/Game of Life.xcodeproj/xcuserdata/berta.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
don't forget to add user data folder to gitignore

(you may also want to make this a shared scheme else it won't turn up when someone else downloads the project)

Reply:
will do, thanks :D